### ISSUE TITLE
Failed to compile "shapes" package in Stanza by Example

### ISSUE THREAD

**Issue by dram** [StanzaOrg/lbstanza-old#13]
When compiling first version of [shapes](http://lbstanza.org/chapter4.html#anchor42) package in "Stanza by Example" with Stanza 0.9.1, it failed with following message:

```
% stanza shapes.stanza shapes-main.stanza -o shapes
FATAL ERROR: No matching branch.
   at core/core.stanza:1462.16
   at core/collections.stanza:351.15
   at core/collections.stanza:358.17
   at core/collections.stanza:408.17
   at core/collections.stanza:287.16
   at core/collections.stanza:298.16
   at compiler/stz-kform.stanza:4680.9
   at compiler/stz-kform.stanza:6056.21
   at compiler/stz-kform.stanza:5703.32
   at compiler/stz-kform.stanza:6052.12
   at compiler/stz-kform.stanza:5706.9
   at compiler/stz-kform.stanza:5807.12
   at compiler/stz-kform.stanza:5807.12
   at compiler/stz-kform.stanza:5402.15
   at core/core.stanza:3099.9
   at core/core.stanza:4101.16
   at compiler/stz-kform.stanza:5391.30
   at compiler/stz-kform.stanza:5647.12
   at core/core.stanza:840.16
   at compiler/stz-kform.stanza:5626.17
   at core/core.stanza:840.16
   at compiler/stz-kform.stanza:5625.14
   at compiler/stz-kform.stanza:5390.9
   at compiler/stz-kform.stanza:5413.24
   at compiler/stz-kform.stanza:6587.24
   at core/core.stanza:3099.9
   at core/core.stanza:4101.16
   at compiler/stz-kform.stanza:6585.27
   at core/core.stanza:840.16
   at compiler/stz-kform.stanza:6596.11
   at compiler/stz-kform.stanza:174.3
   at compiler/stz-compiler.stanza:97.21
   at compiler/stz-main.stanza:206.3
   at compiler/stz-main.stanza:296.21
   at core/core.stanza:3699.22
   at core/core.stanza:840.16
   at core/core.stanza:3698.14
   at core/core.stanza:969.20
   at core/core.stanza:991.31
   at core/core.stanza:855.16
   at compiler/stz-main.stanza:294.10
   at compiler/stz-main.stanza:306.0
```

**Comment by CuppoJava** [comment]
Thanks for the report. I'll look into this.

**Comment by CuppoJava** [comment]
That was a nasty one. Thanks for the bug report! Fixed in 0.9.2.
